Makefile

```
@@ -334,6 +334,17 @@ build/dmadata_table_spec.h: build/$(SPEC)
 build/src/boot/z_std_dma.o: build/dmadata_table_spec.h
 build/src/dmadata/dmadata.o: build/dmadata_table_spec.h
 
+# Dependencies for files including from include/tables/
+# Note z_actor is made dependent on actor_table.h so ACTOR_ID_MAX updates when the table length changes.
+# TODO remove when full header dependencies are used.
+build/src/code/graph.o: include/tables/gamestate_table.h
+build/src/code/object_table.o: include/tables/object_table.h
+build/src/code/z_actor.o: include/tables/actor_table.h
```

```suggestion
build/src/code/z_actor.o: include/tables/actor_table.h # for ACTOR_ID_MAX
```
instead of the above note?